### ISSUE TITLE
python3-rdflib not available

### ISSUE THREAD

**Issue by BlueMax** [falkTX/Carla#1694]
`Python3-rdflib` is not available despite being installed.
Any idea what's wrong?

Arch Linux 5.19.13
Carla df13388
Python-rdflib 6.2.0

![rdflib](https://user-images.githubusercontent.com/1403092/195442493-c328bfbd-7b6f-4209-a6ac-fefc986feacc.png)

**Comment by falkTX** [comment]
No idea, but this will tell you:

```
python3 -c "import ladspa_rdf; print(ladspa_rdf.recheck_all_plugins(None, None, None, None))"
```

**Comment by falkTX** [comment]
Forgot to add a `cd /usr/share/carla/` first

**Comment by BlueMax** [comment]
This throws out a lot of infos indeed:

> $ python3 -c "import ladspa_rdf; print(ladspa_rdf.recheck_all_plugins(None, None, None, None))"
> [{'Type': 2147483648, 'UniqueID': 1922, 'Title': 'Stereo Gate', 'Creator': 'Steve Harris <steveplugin.org.uk>', 'PortCount': 9, 'Ports': [{'Type': 5, 'Hints': 6, 'Label': 'range', 'Default': -90.0, 'Unit': 0, 'ScalePointCount': 2, 'ScalePoints': [{'Value': 0.0, 'Label': 'gate'}, {'Value': 1.0, 'Label': 'bypass'}], 'index': 8}, {'Type': 5, 'Hints': 6, 'Label': 'lf_fc', 'Default': 0.0007, 'Unit': 0, 'ScalePointCount': 0, 'ScalePoints': [], 'index': 1}, {'Type': 5, 'Hints': 6, 'Label': 'threshold', 'Default': -70.0, 'Unit': 0, 'ScalePointCount': 0, 'ScalePoints': [], 'index': 4}, {'Type': 5, 'Hints': 6, 'Label': 'hf_fc', 'Default': 0.49, 'Unit': 0, 'ScalePointCount': 0, 'ScalePoints': [], [...]

**Comment by SpotlightKid** [comment]
> Any idea what's wrong?

Probably nothing. AFAICS, the red X icon is set unconditionally and not dependent on the availability of LADSPA RF support:

https://github.com/falkTX/Carla/blob/4495215cb99bdecb060a5e127a7ea704ab829f7b/source/frontend/pluginlist/pluginlistrefreshdialog.py#L92

It seems this is a remnant of a earlier state of the code?

**Comment by falkTX** [comment]
yeah with the move to C++ on parts of the GUI (as to allow a slow transition vs having to do it all at once) the LRDF stuff is getting removed. it was only used to give extra details for LADSPA plugins, so pretty minor. And most LRDF files that exist out there are wrong/incorrect anyhow.

**Comment by falkTX** [comment]
I actually forgot that was the case..! spending only little snippets of time here and there for carla is what this leads to... sorry for the mess

**Comment by tank-trax** [comment]
I saw this too and have python3-rdflib installed

that command spits out over 10k chars of ladpsa plugin info

so it's just an eyesore? :-)
